examples/experimental/scala-local-helloworld/build.sbt

```diff
@@ -24,6 +24,8 @@ name := "example-scala-local-helloworld"
 organization := "org.sample"
 
 libraryDependencies ++= Seq(
-  "org.apache.predictionio" %% "core" % "0.9.1" % "provided",
-  "org.apache.predictionio" %% "data" % "0.9.1" % "provided",
+  "org.slf4j" % "slf4j-api" % "1.6.1",
+  "org.apache.predictionio" % "apache-predictionio-data_2.10" % "0.10.0-incubating",
```

Comment:
Hi @rajdeepd , sorry for getting back to this late. How about changing this line to
```
"org.apache.predictionio" %% "apache-predictionio-data" % "0.10.0-incubating",
```
Using `%%` will automatically append the proper Scala version suffix.